api/resources_portal/views/material_request.py

```diff
@@ -145,21 +142,25 @@ def has_object_permission(self, request, view, obj):
                 "payment_method",
                 "payment_method_notes",
                 "has_issues",
-                "issues",
                 "requires_action_sharer",
                 "requires_action_requester",
                 "material",
-                "created_at",
-                "updated_at",
             ]
 
         for field in forbidden_fields:
             if field in request.data:
                 attribute = getattr(obj, field)
                 if isinstance(attribute, models.Model):
                     # UUID's can't just be treated as strings for some reason...
```

Comment:
lmao this represented 90% of the issues I ran into when I first started working here. You know my pain